src/state.h

```diff
@@ -126,6 +126,8 @@ struct state_conf {
 	int max_sendto_failures;
 	float min_hitrate;
 	int data_link_size;
+	int default_mode;
+	int no_header_row;
```

Comment:
Should we get rid of `filter_duplicates` and `filter_unsuccessful` from `zconf` as well? They can't be set on the CLI, and I think they always match `default_mode`.